dhis-2/dhis-api/src/main/java/org/hisp/dhis/tracker/imports/validation/ValidationCode.java

```diff
@@ -29,152 +29,141 @@
-  E1113("Enrollment: `{0}`, is already deleted and cannot be modified."),
-  E1114("TrackedEntity: `{0}`, is already deleted and cannot be modified."),
+      "Attribute: `{0}` is mandatory in TrackedEntityType `{1}` but not found in TrackedEntity `{2}`."),
+  E1091("User: `{0}` has no data write access to Program: `{1}`."),
+  E1095("User: `{0}` has no data write access to ProgramStage: `{1}`."),
+  E1096("User: `{0}` has no data read access to Program: `{1}`."),
+  E1099("User: `{0}` has no write access to CategoryOption: `{1}`."),
+  E1100("User: `{0}` lacks 'F_TEI_CASCADE_DELETE' authority to delete TrackedEntity: `{1}`."),
+  E1102("User: `{0}` does not have access to the tracked entity: `{1}` and Program: `{2}`."),
+  E1103("User: `{0}` lacks 'F_ENROLLMENT_CASCADE_DELETE' authority to delete Enrollment : `{1}`."),
+  E1104("User: `{0}` has no data read access to Program: `{1}`, TrackedEntityType: `{2}`."),
+  E1112("AttributeValue: `{0}` is confidential but encryption is not enabled: `{1}`"),
+  E1113("Enrollment: `{0}` is already deleted and cannot be modified."),
+  E1114("TrackedEntity: `{0}` is already deleted and cannot be modified."),
   E1115("Could not find CategoryOptionCombo: `{0}`."),
   E1116("Could not find CategoryOption: `{0}`."),
-  E1117("CategoryOptionCombo does not exist for category combo `{0}` and category options `{1}`."),
-  E1118("Assigned user `{0}` is not valid."),
-  E1119("A Tracker Note with uid `{0}` already exists."),
-  E1120("ProgramStage `{0}` does not allow user assignment"),
+  E1117("CategoryOptionCombo not found for CategoryCombo `{0}` and CategoryOptions `{1}`."),
+  E1118("Assigned User `{0}` is not valid."),
+  E1119("A Note with UID `{0}` already exists."),
+  E1120("ProgramStage `{0}` does not allow user assignment."),
   E1121("Missing required trackedEntity property: `{0}`."),
   E1122("Missing required enrollment property: `{0}`."),
   E1123("Missing required event property: `{0}`."),
   E1124("Missing required relationship property: `{0}`."),
-  E1125("Value `{0}` is not a valid option code in option set `{1}`"),
-  E1126("Not allowed to update Tracked Entity property: {0}."),
-  E1127("Not allowed to update Enrollment property: {0}."),
-  E1128("Not allowed to update Event property: {0}."),
-  E1300("Generated by program rule (`{0}`) - `{1}`"),
-  E1301("Generated by program rule (`{0}`) - Mandatory DataElement `{1}` is not present"),
-  E1302("DataElement `{0}` is not valid: `{1}`"),
-  E1303("Mandatory DataElement `{0}` is not present"),
-  E1304("DataElement `{0}` is not a valid data element"),
-  E1305("DataElement `{0}` is not part of `{1}` program stage"),
-  E1306("Generated by program rule (`{0}`) - Mandatory Attribute `{1}` is not present"),
+  E1125("Value `{0}` is not a valid option code in OptionSet `{1}`."),
+  E1126("Not allowed to update trackedEntity property: {0}."),
```

Comment:
In other places `TrackedEntity`, `Enrollment` and `Event` start with a capital letter. Even when they are a property.